src/tax-lot/tax-lot.service.ts

```diff
@@ -57,4 +57,11 @@ export class TaxLotService {
       zoningDistrictClasses,
     };
   }
+
+  async findTaxLotTilesets(params: { z: number; x: number; y: number }) {
+    const url = await this.taxLotRepository.findTaxLotTilesets(params);
+    return {
```

Comment:
Delegate the formatting of the redirect url into the service under the argument that formatting is business logic